src/components/AstronautsPage/components/AstronautsList/components/AstronautsItem/components/AstronautsItemContent/AstronautsItemContent.tsx

```diff
@@ -1,13 +1,24 @@
 import React from 'react';
 import { CardContent, Typography } from '@mui/material';
-import { astronautsData } from '../../../../../../mocks/astronautsData';
+// import { astronautsData } from '../../../../../../mocks/astronautsData';
 import './AstronautsItemContent.scss';
 
-export const AstronautsItemContent = () => (
+interface AstronautsItemContentName {
+  name: string;
+}
+
+interface AstronautsItemContent extends AstronautsItemContentName {
+  agency: AstronautsItemContentName;
+}
+
+export const AstronautsItemContent = ({
+  name,
+  agency,
+}: AstronautsItemContent) => (
```

Comment:
For what there are two interface and why we cant combine them into one. I guess interface must have name with ....Props in the end

Reply:
No, we can't. Because AstronautsItemContent have own field name: string and agency have object name: string.
But i still left one interface.
...Props added in the end of interface name.